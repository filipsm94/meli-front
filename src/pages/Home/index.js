import React from 'react';

const HomePage = () => <h1>Bienvenido a mercado libre</h1>

export default HomePage;