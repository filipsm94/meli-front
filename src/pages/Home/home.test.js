import { render, screen } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import Home from './index';

describe('Home Component',()=>{
  test('renders', () => {
    
    const {container} = render(<Home />);
    expect(container).toMatchSnapshot();
  });

})
