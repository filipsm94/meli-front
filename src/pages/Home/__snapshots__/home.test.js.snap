// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Component renders 1`] = `
<div>
  <h1>
    Bienvenido a mercado libre
  </h1>
</div>
`;
